### PR TITLE
Update channels from Spain

### PR DESCRIPTION
**Pull request by sguinetti**
This PR edit RTVE channels, and add other local channels.

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -2887,11 +2887,13 @@ Canal222.mx,Canal 22.2,,,MX,,,c/MX,spa,,FALSE,,,,https://www.canal22.org.mx/,htt
...
+Canal24Horas.es,Canal 24 Horas,24h,RTVE,ES,,,c/ES,spa;eng,,FALSE,,,,http://www.rtve.es/alacarta/tve/24-horas/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
+Canal24Canarias.es,Canal 24 Horas Canarias,24h,RTVE,ES,ES-CN,,s/ES-CN,spa;eng,,FALSE,,,,https://www.rtve.es/play/videos/directo/tve-canarias/24-h-can/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
+Canal24Catalunya.es,Canal 24 Horas Catalunya,24h,RTVE,ES,ES-CT,,s/ES-CT,cat;eng,,FALSE,,,,https://www.rtve.es/play/videos/directo/tve-catalunya/24h-cat/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
+Canal24HorasInternacional.es,Canal 24 Horas Internacional,24h,RTVE,ES,,,r/AMER;r/EUR,spa;eng,,FALSE,,,,https://www.rtve.es/television/tve-internacional,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
```
Comment (freearhey) [inline data/channels.csv:2893]:
```suggestion
Canal24HorasInternacional.es,Canal 24 Horas Internacional,,RTVE,ES,,,r/AMER;r/EUR,spa;eng,,FALSE,,,,https://www.rtve.es/television/tve-internacional,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
```

**Review by freearhey — CHANGES_REQUESTED** [review]

File: data/channels.csv [hunk 2]
```diff
@@ -2887,11 +2887,13 @@ Canal222.mx,Canal 22.2,,,MX,,,c/MX,spa,,FALSE,,,,https://www.canal22.org.mx/,htt
 Canal22Internacional.mx,Canal 22 Internacional,,,MX,,,c/MX;c/US,spa,,FALSE,,,,https://www.canal22.org.mx/,https://i.imgur.com/wVIUIee.png
 Canal22MetropolitanayNacional.mx,Canal 22 Metropolitana y Nacional,,,MX,,,c/MX,spa,,FALSE,,,,https://www.canal22.org.mx/,https://cdn.mitvstatic.com/channels/mx_canal-22-conaculta_m.png
 Canal22Nacional.mx,Canal 22 Nacional,,,MX,,,c/MX,spa,,FALSE,,,,https://www.canal22.org.mx/,https://imagenes.gatotv.com/logos/canales/oscuros/22_de_mexico-mediano.png
-Canal24Horas.es,Canal 24 Horas,,,ES,,,c/ES,spa;eng,,FALSE,,,,http://www.rtve.es/alacarta/tve/24-horas/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
-Canal24HorasInternacional.es,Canal 24 Horas Internacional,,,ES,,,r/AMER;r/EUR,spa;eng,,FALSE,,,,https://www.rtve.es/television/tve-internacional,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
+Canal24Horas.es,Canal 24 Horas,24h,RTVE,ES,,,c/ES,spa;eng,,FALSE,,,,http://www.rtve.es/alacarta/tve/24-horas/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
```
[the remark this answers is not in the document]
Comment (sguinetti) [reply data/channels.csv:2890]: Done

**Comment by sguinetti** [comment]
Updated: New channels from ES-CT.

File: data/channels.csv [hunk 3]
```diff
@@ -2877,21 +2878,24 @@ Canal2.co,Canal 2,,,CO,,,c/CO,spa,general,FALSE,,,,,https://cdn.colombia.com/can
...
 Canal22Internacional.mx,Canal 22 Internacional,,,MX,,,c/MX;c/US,spa,,FALSE,,,,https://www.canal22.org.mx/,https://i.imgur.com/wVIUIee.png
 Canal22MetropolitanayNacional.mx,Canal 22 Metropolitana y Nacional,,,MX,,,c/MX,spa,,FALSE,,,,https://www.canal22.org.mx/,https://cdn.mitvstatic.com/channels/mx_canal-22-conaculta_m.png
 Canal22Nacional.mx,Canal 22 Nacional,,,MX,,,c/MX,spa,,FALSE,,,,https://www.canal22.org.mx/,https://imagenes.gatotv.com/logos/canales/oscuros/22_de_mexico-mediano.png
-Canal24Horas.es,Canal 24 Horas,,,ES,,,c/ES,spa;eng,,FALSE,,,,http://www.rtve.es/alacarta/tve/24-horas/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
-Canal24HorasInternacional.es,Canal 24 Horas Internacional,,,ES,,,r/AMER;r/EUR,spa;eng,,FALSE,,,,https://www.rtve.es/television/tve-internacional,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
+Canal24Horas.es,Canal 24 Horas,24h,RTVE,ES,,,c/ES,spa;eng,,FALSE,,,,http://www.rtve.es/alacarta/tve/24-horas/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
```
Comment (freearhey) [inline data/channels.csv:2892]:
```suggestion
Canal24Horas.es,Canal 24 Horas,,RTVE,ES,,,c/ES,spa;eng,,FALSE,,,,http://www.rtve.es/alacarta/tve/24-horas/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
```

File: data/channels.csv [hunk 4]
```diff
@@ -2877,21 +2878,24 @@ Canal2.co,Canal 2,,,CO,,,c/CO,spa,general,FALSE,,,,,https://cdn.colombia.com/can
...
+Canal24Canarias.es,Canal 24 Horas Canarias,24h,RTVE,ES,ES-CN,,s/ES-CN,spa;eng,,FALSE,,,,https://www.rtve.es/play/videos/directo/tve-canarias/24-h-can/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
```
Comment (freearhey) [inline data/channels.csv:2893]:
```suggestion
Canal24Canarias.es,Canal 24 Horas Canarias,,RTVE,ES,ES-CN,,s/ES-CN,spa;eng,,FALSE,,,,https://www.rtve.es/play/videos/directo/tve-canarias/24-h-can/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
```

File: data/channels.csv [hunk 5]
```diff
@@ -2877,21 +2878,24 @@ Canal2.co,Canal 2,,,CO,,,c/CO,spa,general,FALSE,,,,,https://cdn.colombia.com/can
...
+Canal24Catalunya.es,Canal 24 Horas Catalunya,24h,RTVE,ES,ES-CT,,s/ES-CT,cat;eng,,FALSE,,,,https://www.rtve.es/play/videos/directo/tve-catalunya/24h-cat/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
```
Comment (freearhey) [inline data/channels.csv:2894]:
```suggestion
Canal24Catalunya.es,Canal 24 Horas Catalunya,,RTVE,ES,ES-CT,,s/ES-CT,cat;eng,,FALSE,,,,https://www.rtve.es/play/videos/directo/tve-catalunya/24h-cat/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
```

File: data/channels.csv [hunk 6]
```diff
@@ -2877,21 +2878,24 @@ Canal2.co,Canal 2,,,CO,,,c/CO,spa,general,FALSE,,,,,https://cdn.colombia.com/can
...
+Canal24HorasInternacional.es,Canal 24 Horas Internacional,24h,RTVE,ES,,,r/AMER;r/EUR,spa;eng,,FALSE,,,,https://www.rtve.es/television/tve-internacional,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
```
Comment (freearhey) [inline data/channels.csv:2895]:
```suggestion
Canal24HorasInternacional.es,Canal 24 Horas Internacional,,RTVE,ES,,,r/AMER;r/EUR,spa;eng,,FALSE,,,,https://www.rtve.es/television/tve-internacional,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
```

**Review by freearhey — CHANGES_REQUESTED** [review]
`native_name` is used for the name of the channel in the original language if it is different from the name

**Comment by sguinetti** [comment]
I am going to edit it manually, it does not allow me to remove the text "24h".

**Comment by sguinetti** [comment]
Updated with edit for TV5Monde and ZooMoo broadcasting.

File: data/channels.csv [hunk 7]
```diff
@@ -29067,4 +29074,24 @@ Vivamovil.es,Vivamóvil,,,ES,ES-AN,,s/ES-AN,spa,,FALSE,,,,https://vivamovil.tv/e
...
+TVBergueda.es,TV Berguedà,,,ES,ES-CT,Bergueda,s/ES-CT,cat,,FALSE,,,,https://tvbergueda.alacarta.cat/,https://www.alacarta.cat/api/assets/getFile/public/media/000/793/324/logoalfa.png
+RTVElVendrell.es,RTV El Vendrell,,,ES,ES-CT,El Vendrell,s/ES-CT,cat,,FALSE,,,,https://www.rtvelvendrell.cat/directes/televisio/,https://www.rtvelvendrell.cat/wp-content/uploads/2020/09/logo-rtv-reduit-1.png
+VOTV.es,VOTV,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,http://votv.alacarta.cat/votv-en-directe,https://i.imgur.com/1g2Z64L.png
+PieraTV.es,Piera TV,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,http://08-produccions.cat/pieratv/,https://yt3.ggpht.com/WtsIUXtvmo0QQFYhj9qbxpttucZF0ZElx8oju9r7ISrKfn9GHBuRR7s9JN4YRnzvbLr6GiUtWQ=s200-c-k-c0x00ffffff-no-rj
+RTVCCardedeu.es,RTVC,,,ES,ES-CT,Cardedeu,s/ES-CT,cat,,FALSE,,,,https://www.rtvc.cat/tele/,https://www.rtvc.cat/wp-content/uploads/2020/06/LOGO_RTVC_WEB.png
+Vallesvisio.es,Vallès Visió,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,https://www.vallesvisio.cat/directe/,https://www.vallesvisio.cat/wp-content/uploads/cropped-Logo-Valles-Visio-1.png
```
Comment (freearhey) [inline data/channels.csv:29094]:
```suggestion
VallesVisio.es,Vallès Visió,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,https://www.vallesvisio.cat/directe/,https://www.vallesvisio.cat/wp-content/uploads/cropped-Logo-Valles-Visio-1.png
```

File: data/channels.csv [hunk 8]
```diff
@@ -29067,4 +29074,24 @@ Vivamovil.es,Vivamóvil,,,ES,ES-AN,,s/ES-AN,spa,,FALSE,,,,https://vivamovil.tv/e
...
+TVLHospitalet.es,TV L'Hospitalet,,,ES,ES-CT,L'Hospitalet,s/ES-CT,cat,,FALSE,,,,https://lhdigital.cat/web/digital-h/en-directe,https://upload.wikimedia.org/wikipedia/commons/thumb/e/e4/Logotip_TV_L%27H.svg/400px-Logotip_TV_L%27H.svg.png
+TVBergueda.es,TV Berguedà,,,ES,ES-CT,Bergueda,s/ES-CT,cat,,FALSE,,,,https://tvbergueda.alacarta.cat/,https://www.alacarta.cat/api/assets/getFile/public/media/000/793/324/logoalfa.png
+RTVElVendrell.es,RTV El Vendrell,,,ES,ES-CT,El Vendrell,s/ES-CT,cat,,FALSE,,,,https://www.rtvelvendrell.cat/directes/televisio/,https://www.rtvelvendrell.cat/wp-content/uploads/2020/09/logo-rtv-reduit-1.png
+VOTV.es,VOTV,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,http://votv.alacarta.cat/votv-en-directe,https://i.imgur.com/1g2Z64L.png
+PieraTV.es,Piera TV,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,http://08-produccions.cat/pieratv/,https://yt3.ggpht.com/WtsIUXtvmo0QQFYhj9qbxpttucZF0ZElx8oju9r7ISrKfn9GHBuRR7s9JN4YRnzvbLr6GiUtWQ=s200-c-k-c0x00ffffff-no-rj
+RTVCCardedeu.es,RTVC,,,ES,ES-CT,Cardedeu,s/ES-CT,cat,,FALSE,,,,https://www.rtvc.cat/tele/,https://www.rtvc.cat/wp-content/uploads/2020/06/LOGO_RTVC_WEB.png
```
Comment (freearhey) [inline data/channels.csv:29093]:
```suggestion
RTVC.es,RTVC,,,ES,ES-CT,Cardedeu,s/ES-CT,cat,,FALSE,,,,https://www.rtvc.cat/tele/,https://www.rtvc.cat/wp-content/uploads/2020/06/LOGO_RTVC_WEB.png
```

File: data/channels.csv [hunk 9]
```diff
@@ -29067,4 +29074,24 @@ Vivamovil.es,Vivamóvil,,,ES,ES-AN,,s/ES-AN,spa,,FALSE,,,,https://vivamovil.tv/e
...
+Vallesvisio.es,Vallès Visió,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,https://www.vallesvisio.cat/directe/,https://www.vallesvisio.cat/wp-content/uploads/cropped-Logo-Valles-Visio-1.png
+ETV.es,etv,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,http://www.etv.cat/,https://www.alacarta.cat/api/assets/getFile/public/media/000/557/386/etv_500trtm3.png
```
Comment (freearhey) [inline data/channels.csv:29095]:
```suggestion
etv.es,etv,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,http://www.etv.cat/,https://www.alacarta.cat/api/assets/getFile/public/media/000/557/386/etv_500trtm3.png
```

**Review by freearhey — CHANGES_REQUESTED** [review]

File: data/channels.csv (hunk 8 above)
Comment (sguinetti) [reply data/channels.csv:29093]: There are two channels similar to RTVC. One is in the Canarias and the other in Catalonia (see the ".es" and ".cat" from the official website).

File: data/channels.csv (hunk 9 above)
Comment (sguinetti) [reply data/channels.csv:29095]: Done

File: data/channels.csv (hunk 7 above)
Comment (sguinetti) [reply data/channels.csv:29094]: Done

File: data/channels.csv (hunk 8 above)
Comment (freearhey) [reply data/channels.csv:29093]: Probably best to change the name then, so there is no confusion.
```suggestion
RTVCCardedeu.es,RTVC Cardedeu,,,ES,ES-CT,Cardedeu,s/ES-CT,cat,,FALSE,,,,https://www.rtvc.cat/tele/,https://www.rtvc.cat/wp-content/uploads/2020/06/LOGO_RTVC_WEB.png
```

Comment (sguinetti) [reply data/channels.csv:29093]: Done. Thank you for this idea.

**Review by freearhey — APPROVED** [review]

**Pull request merged**